### DISCUSSION TITLE
css :root

### DISCUSSION THREAD

**Discussion by aviv82** [lab-antwerp-1/agile-group-1#39] [General]
:root {
  --header-font-family: eu;
  --wine-light-color: #f3e2e3;
  --wine-mid1-color: #8a5658;
  --wine-mid2-color: #a04048;
  --wine-dark-color: #582327;
}

**Comment by denrique-alvarez** [comment]
:root {
    --header-font-family: eu;
    --paragraph-font: something to decide;
    --wine-light-color: #f3e2e3;
    --wine-mid1-color: #8a5658;
    --wine-mid2-color: #a04048;
    --wine-dark-color: #582327;
}

html {
    box-sizing: border-box;
    font-size: 62.5%;
    /* 1rem = 10px */
}

*,
*:before,
*:after {
    box-sizing: inherit;
    /* Keeps the size of the elements even when changing 'margin', 'padding', etc. */
}

body {
    font-family: var();
    font-size: 2rem;
    line-height: 1.8;
    text-align: center;
    /* General text distribution */
}


/* Globals */

.container {
    width: min(90%, 120rem);
    margin: 0 auto;
    padding: 1rem;
    background-color: var(--wine-light-color);
}

a {
    text-decoration: none;
    /* Removes the underlining of the links */
}

h1,
h2,
h3,
h4 {
    font-family: var(--header-font-family);
    line-height: 1.3;
}

h1 {
    font-size: 4.8rem;
}

h2 {
    font-size: 4rem;
}

h3 {
    font-size: 3.2rem;
}

h4 {
    font-size: 2.8rem;
}

img {
    max-width: 100%;
}

.no-margin {
    margin: 0;
}

.no-padding {
    padding: 0;
}

.center-text {
    text-align: center;
}

/* HEADER */

/* HOMEPAGE */

/* TOP 5 */

.topImg {
    display: block;
    margin: 0 auto;
    width: max 80%;
}

.topText {
    text-align: justify;
    width: 85%;
    margin: 0 auto;
}


/* PREPARATION */

/* FOOTER */

**Reply by mauricemat** [reply]
Thank you @denrique-alvarez ,
